index.html

```diff
@@ -5,11 +5,16 @@
     <link rel="stylesheet" type="text/css" href="style.css" />
   </head>
   <body>
+    <nav>
+      <a href="index.html" class="a">Home</a>
+      <a href="numbers.html" class="a">Guess the number game</a>
+      <a href="contact.html" class="a">Contact</a>
+      <a href=""></a>
     <h1>Welcome to Jeff McCord's About Me Page</h1>
     <p class="intro">As a student, I want to create an About Me page to demonstrate my skills with HTML and JavaScript.</p>
     <p>I currently work for a Canadian company called HERO.  They manufacture paint auto-tinters,  to which I install and repair. I cover <b>6</b> states, Washington, Oregon, Montana, Idaho, Alaska, and occasionally, Wyoming. The region I amm in, is one of the the slowest in the country, and I’ve gotten bored. So I’m here, at CodeFellows, to change careers to something that I want to do, not something that I need to do in order to live.   </p>
     <p>In my free time, I hang out with my bf, friends, and my mother.  However, most of the time I’m doing other things to keep my mind busy, such as video games, designing and building a new “jack in the box”, and now school. I’m also getting supplies together to create my own <em>whiskey</em>.  It’s a birthday present for my nephew, and for my sister and her husband, when he turns 21.  But if I’m going to let it age for 20 years, it may be a few years late.  I already have the storing place set.  Just need to build the distiller and create the mash.  Also still debating on what size of barrels I’m going to use.  Since they come in many sizes.  If I have enough mash, I’m more than happy to make a few smaller barrels for people to have and let sit, or they can drink it straight up like moonshine.</p>
-
+<a href="sub.html" class="a">Click here to guess the cities I've lived in!</a>
```

Comment:
watch your indentation here, looks like your `<a>` tag and `<body>` tags are little off.